### PR TITLE
Bug 5915906 vertical stacked chart colors of legends is not displayed in high contrast

### PR DESCRIPTION
**Pull request by ankityadav4**
## Current Behavior
LegendBefore

## New Behavior

changedColor
CustomAccessbility
callout
styled

## Related Issue(s)

Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/0915feeaa4278c32a7818b67b159ed207d1031fc'>0915feeaa4278c32a7818b67b159ed207d1031fc</a></sub>
<div id="bundleSizeComment9423"/>

File: change/@fluentui-react-charting-6612157f-b40c-447b-80fa-57671369d12c.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "type": "patch",
+  "comment": {
```
Comment (AtishayMsft) [inline change/@fluentui-react-charting-6612157f-b40c-447b-80fa-57671369d12c.json:3]: Add an appropriate comment.

File: packages/react-charting/src/components/CommonComponents/CartesianChart.styles.ts [hunk 2]
```diff
@@ -138,18 +143,33 @@ export const getStyles = (props: ICartesianChartStyleProps): ICartesianChartStyl
     calloutlegendText: {
       ...fonts.small,
       lineHeight: '16px',
+      selectors: {
+        [HighContrastSelector]: {
+          color: 'rgb(255, 255, 255)',
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/CommonComponents/CartesianChart.styles.ts:148]: Could you explain this rgb 255 color value. Why do you need to hardcode.

Comment (ankityadav4) [reply packages/react-charting/src/components/CommonComponents/CartesianChart.styles.ts:148]: The color of the text in the tool tip was getting very dim after applying the high contrast theme so to make it brighter i did that. Should i use the theme instead.

Comment (AtishayMsft) [reply packages/react-charting/src/components/CommonComponents/CartesianChart.styles.ts:148]: Is the tool tip text customizable by the user? If yes it should be taken from props.

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**